web_app/config/routes.rb

```diff
@@ -5,8 +5,9 @@
     sessions: "users/sessions",
     omniauth_callbacks: "users/omniauth_callbacks"
   }
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :policies
```

Comment:
:art:  Aqui você também poderia limitar as rotas para apenas a que você utiliza
```suggestion
    resource :policies, only: [:index, :new, :create, :show]
```

ou excluir as que não utiliza
```suggestion
  resources :policies, except: [ :edit, :destroy, :update]
```

Reply:
verdade, eu coloquei aí por que na minha falta de organização acabei colocando rotas que irei utilizar nos próximos passos, ao invés de apenas as que estou utilizando atualmente, já que pretendia adicionar agora o update, destroy e edit de um policy